neural_modelling/src/neuron/plasticity/synapse_dynamics_static_impl.c

```diff
@@ -47,3 +47,25 @@ void synapse_dynamics_print_plastic_synapses(
 uint32_t synapse_dynamics_get_plastic_pre_synaptic_events() {
     return 0;
 }
+
+bool find_plastic_neuron_with_id(uint32_t id, address_t row, structural_plasticity_data_t *sp_data){
+    use(id);
+    use(row);
+    use(sp_data);
+    return false;
+}
+
+bool remove_plastic_neuron_at_offset(uint32_t offset, address_t row){
```

Comment:
I hope you noticed that nothing in this file, or synapse_dynamics_stdp_mad_impl.c has any sphinx docs.

Reply:
i know, that's a problem we should have caught with Jamie when he did them. alas that got through review......given we basically didn't review it. your stuff will be better supported because of these new comments.

your also welcome to comment the jamie stuff if you understand it enough